Header header
uint16 ch1_rol
uint16 ch2_pit
uint16 ch3_thr
uint16 ch4_yaw
uint16 ch5_aux1
uint16 ch6_aux2
uint16 ch7_aux3
uint16 ch8_aux4
uint16 ch9_aux5
uint16 ch10_aux6

